.spacelift/config.yml

```
@@ -1,5 +1,5 @@
 version: 1
-module_version: 1.8.0
+module_version: 1.9.0
```

I would consider that a fix for version 1.8.0. It does not introduce any new features.

```suggestion
module_version: 1.8.1
```

---

Updated